unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShapeLoaderController.cs

```diff
@@ -8,7 +8,7 @@
 
 public interface INFTShapeLoaderController
 {
-    BoxCollider nftCollider { get; }
+    Collider nftCollider { get; }
```

Comment:
why are we changing this?

Reply:
Since we don't need a specific thing of a BoxCollider, this was done to be more flexible in the future if we want to implement NFTShapes with another collider. Anyway, I have undone it